frontend/src/components/course/components/components.tsx

```diff
@@ -0,0 +1,3 @@
+export { AddUpdateCategoryModal } from './add-categories-modal/add-categories-modal';
```

Comment:
Let's make file and component naming consistent + EditCategoryModal may be more suitable naming